packages/components/image/src/Image.vue

```diff
@@ -0,0 +1,115 @@
+<template>
+  <div class="ix-image">
+    <img
+      v-if="imageStatus !== 'error'"
+      :class="['ix-image-img', preview ? 'image-preview' : '']"
+      :src="src"
+      :style="{ width: imageWidth, height: imageHeight, objectFit: objectFit }"
+      :alt="alt"
+      @error="onError"
+      @load="onLoaded"
+      @click="onClick"
+    />
+    <div
+      v-else-if="imageStatus === 'error'"
+      class="image-error"
+      :style="{
+        width: imageWidth,
+        height: imageHeight,
+        objectFit: objectFit,
+        background: `url(${fallback || imageConfig.fallback})`,
+      }"
+    ></div>
+    <template v-if="imageStatus !== 'error' && preview">
+      <img-preview v-if="isShowPreview" :previewSrc="src" @close="onClose"></img-preview>
+    </template>
+  </div>
+</template>
+<script lang="ts">
+import { defineComponent, computed, ref, watchEffect, watch } from 'vue'
+import { ImageProps } from './types'
+import ImgPreview from './ImgPreview.vue'
+import { useGlobalConfig } from '@idux/components/core/config'
+
+export default defineComponent({
+  name: 'IxImage',
+  components: { ImgPreview },
+  props: {
+    src: {
+      type: String,
+      default: '',
+    },
+    width: {
+      type: [String, Number],
+      default: null,
+    },
+    height: {
+      type: [String, Number],
+      default: null,
+    },
+    preview: {
+      type: Boolean,
+      default: false,
+    },
+    fallback: {
+      type: String,
+      default: '',
+    },
+    alt: String,
+    objectFit: {
+      type: String,
+      default: 'fill',
+    },
+  },
+  setup(props: ImageProps, { emit }) {
+    const isShowPreview = ref(false)
+    const imageConfig = useGlobalConfig('image')
+    const imageWidth = computedImg(props, imageConfig, 'width')
+    const imageHeight = computedImg(props, imageConfig, 'height')
+    const imageStatus = ref('loading')
+    const onError = () => {
+      imageStatus.value = 'error'
+    }
+    const onLoaded = () => {
+      imageStatus.value = 'loaded'
+    }
+    const onClick = () => {
+      isShowPreview.value = true
+    }
+    const onClose = () => {
+      isShowPreview.value = false
+    }
+    watchEffect(() => {
+      const status = imageStatus.value
+      emit('statusChange', status)
+    })
+    watch(
+      () => props.src,
+      () => {
+        imageStatus.value = 'loading'
+      },
+    )
+    return {
+      imageWidth,
+      imageHeight,
+      onLoaded,
+      onError,
+      onClick,
+      isShowPreview,
+      onClose,
+      imageStatus,
+      imageConfig,
+    }
+  },
+})
+const computedImg = (props: ImageProps, imageConfig: ImageProps, type: 'width' | 'height') => {
+  let imageRect
+  return computed(() => {
+    if (props[type] == null) {
+      imageRect = typeof imageConfig[type] === 'number' ? `${imageConfig[type]}px` : imageConfig[type]
+    }
+    imageRect = typeof props[type] === 'number' ? `${props[type]}px` : props[type]
+    return imageRect || '100%'
```

Comment:
```suggestion
  const size = props[type] ?? imageConfig[type]
  return toCssPixel(size)
```

`toCssPixel` 在 cdk 包中